cruise.umple/test/cruise/umple/compiler/UmpleParserTracerTest.java

```diff
@@ -1144,7 +1146,7 @@ private void assertParse(String inCode, String expectedOutput)
 
 	private void assertParse(String filename, String expectedOutput, boolean expected)
 	{
-		UmpleFile file = new UmpleFile(pathToInput,filename);
+    File file = new File(pathToInput,filename);
```

Comment:
This one looks like it's actually because the tests were written with tabs instead of 2 space indentations.
Should I just change mine to tabs? Otherwise it's changing the entire file to use spaces, which would be consistent with the rest of Umple, but would attribute every line changed to me.